### PR TITLE
Choose me! This code is perfect

### PR DESCRIPTION
**Pull request by akshaybhandariDFT**
## Proposed changes

I made this code 100000000x better. 

## Types of changes

Made it better.

- [X ] Bugfix (change which fixes an issue)
- [ X] New feature (change which adds functionality)
- [ X] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ X] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [X ] I have checked that my changes have not broken any other functionality
- [ X] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [X ] All of your unit tests pass
- [X ] Your code is clear and easy to understand
- [ X] You have documented any new features
- [ ] They can run the code you have written
- [ ] Your changes do not break any other functionality

## Who is reviewing this PR?

@DIPAD-Fran-Bryden

### CODE REVIEW

File: text_file.txt [hunk 1]
```diff
@@ -3,3 +3,8 @@ This is a random text file. You can add and remove things to it to test how git
 
 This is some text.
 This is some other text.
+
+
+Hi this is Akshay (Feb 2024)
+
```
Comment (Zavinta-K) [inline text_file.txt:9]: Hi Akshay. This is Zavinta

File: text_file.txt [hunk 2]
```diff
@@ -3,3 +3,8 @@ This is a random text file. You can add and remove things to it to test how git
...
+And here is Archie too!
```
Comment (Zavinta-K) [inline text_file.txt:10]: Hello

**Review by Zavinta-K — APPROVED** [review]
Actually, this code is perfect :)

File: text_file.txt (hunk 1 above)
Comment (Seymiotics) [inline text_file.txt:9]: great point, very concise

File: text_file.txt [hunk 3]
```diff
@@ -3,3 +3,8 @@ This is a random text file. You can add and remove things to it to test how git
 
 This is some text.
 This is some other text.
+
+
+Hi this is Akshay (Feb 2024)
```
Comment (Seymiotics) [inline text_file.txt:8]: nice to meet you!

**Review by Seymiotics — APPROVED** [review]

**Pull request closed without merging**